README.md

```diff
@@ -598,8 +598,59 @@ The buildpack includes a variety of telemetry agents that can be configured to c
+The first one collects container and database metrics, while the second collects metrics related to the Mendix Runtime.
+
+To enable the integration you must provide the following variables:
+
+| Environment variable    | Value example                                  | Default                  | Description                                                                                                                            |
+|-------------------------|------------------------------------------------|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| `NEW_RELIC_LICENSE_KEY` | `api_key`                                      | -                        | License Key or API Key ([docs](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/))                                    |
+| `NEW_RELIC_METRICS_URI` | `https://metric-api.eu.newrelic.com/metric/v1` | -                        | Metrics endpoint API ([docs](https://docs.newrelic.com/docs/data-apis/ingest-apis/metric-api/report-metrics-metric-api/#api-endpoint)) |
+| `NEW_RELIC_LOGS_URI`    | `https://log-api.eu.newrelic.com/log/v1`       | -                        | Logs endpoint API ([docs](https://docs.newrelic.com/docs/logs/log-api/introduction-log-api/))                                          |
+| `NEW_RELIC_APP_NAME`    | `MyApp`                                        | application domain name  | Mendix App environment ID                                                                                                              |
+
+:warning: For the first usage of the New Relic integration, the Mendix app should be redeployed after setting the variables up.
+
+Custom tags
+You can also set up custom tags in the following format key:value. We recommend that you add the following custom tags:
+
+app:{app_name} – this enables you to identify all logs sent from your app (for example, app:customermanagement)
+env:{environment_name} – this enables you to identify logs sent from a particular environment so you can separate out production logs from test logs (for example, env:accp)
+
+#### Metadata
+
+In addition to the runtime application logs, the following JSON-formatted metadata is automatically sent to New Relic:
+
+* `environment_id` - unique identifier of the environment;
+* `instance_index` - number of the application instance;
+* `hostname` - name of the application host;
+* `application_name` - default application name, retrieved from domain name;
+* `model_version` - model version of the Mendix runtime;
+* `runtime_version` - version of the Mendix runtime.
+
+The same values are also provided with the custom metrics pushed to New Relic.
```

Comment:
This metadata is available with the NR agent metrics too right ?

Reply:
`model_version` and `runtime_version` currently are not available for them. I can try to re-phrase it.
But also they don't show in the same way as the custom metrics. For custom metrics, they show up as columns when we query the metrics, the metrics collected by the agent don't have these columns, but looking in the APM & Services, you can see the tags that are set.

Reply:
Okie got it. Ignore the comment then.